src/doc/unstable-book/src/language-features/trace-macros.md

```diff
@@ -0,0 +1,39 @@
+# `trace_macros`
+
+The tracking issue for this feature is [#29598].
+
+[#29598]: https://github.com/rust-lang/rust/issues/29598
+
+------------------------
+
+With `trace_macros` you can trace the expansion of macros in your code.
+
+## Examples
+
+```rust
+#![feature(trace_macros)]
+
+fn main() {
+    trace_macros!(true);
+    println!("Hello, Rust!");
+    trace_macros!(false);
+}
+```
+
+The `cargo build` output:
+
+```
```

Comment:
Looks like travis choked on the "results" block, because code blocks default to being Rust code. make this ```text instead and this should get past that error.

Reply:
Done and rebased!

Reply:
Perfect! Let's see what travis says now.